components/twitter/actions/add-user-to-list/add-user-to-list.mjs

```diff
@@ -0,0 +1,54 @@
+import twitter from "../../twitter.app.mjs";
+
+export default {
+  key: "twitter-add-user-to-list",
+  name: "Add User To List",
+  description: "Add a member to a list. The authenticated user must own the list to be able to add members to it",
```

Comment:
Just to confirm, should this apply to all sources as well?

Reply:
@michelle0927 I don't think so

Reply:
Ok, thanks. For my own reference, why actions and not sources?

Reply:
In my view. It's not worth doing it in the sources because sometimes we do not have real webhooks, we use actions to list something and do some diff to generate an event source. Maybe it's worth linking the docs if we are consuming a real webhook.
Does it make sense for you?
Cc: @dylburger @js07 @dannyroosevelt

Reply:
What I'm gathering is that if an action or a source gets its results primarily from one particular API endpoint, we should include the documentation to it. If an action or source uses a variety of endpoints to get the result, we probably don't need to include a link. Does that sound right?

Reply:
that seems reasonable to me